simgui-window.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
The other file can be committed, but don't commit this one. It changes too often.